app/src/main/java/ch/epfl/sdp/blindwar/login/UserNewInfoActivity.kt

```diff
@@ -91,12 +94,21 @@ class UserNewInfoActivity : AppCompatActivity() {
         }
     }
 
-//    override fun onBackPressed() { // TODO: when returning on SplashSreenActivity, not OK...
-//        super.onBackPressed()
-//        AuthUI.getInstance().signOut(this)
-//        AuthUI.getInstance().delete(this)
-//    }
+    /**
+     * To avoid crashing the app, if the back button is pressed, user will log out
+     *
+     */
+    override fun onBackPressed() {
+        //this.moveTaskToBack(true);
+        auth.signOut()
```

Comment:
I think that this should bu true only if user is a new one (and thus, we should also delete the newly created account on firebase)

Reply:
yeah, that's true, I'll do a separate PR for that